engine/table/src/main/java/io/deephaven/engine/table/impl/lang/QueryLanguageParser.java

```diff
@@ -2430,6 +2458,52 @@ public Class<?> visit(MethodCallExpr n, VisitArgs printer) {
                 typeArguments);
     }
 
+    private Optional<CastExpr> makeCastExpressionForPyCallable(Class<?> retType, MethodCallExpr callMethodCall) {
```

Comment:
Looking through this list, I see that timestamps are missing. Are there any timestamp conversions that we should be transparently handling? Probably, but it may not be easy. I don't know.